devicon.json

```diff
@@ -6266,6 +6266,25 @@
         "color": "#149B9A",
         "aliases": []
     },
+    {
+        "name": "memcached",
+        "altnames": [],
+        "tags": [
+            "data",
+            "database",
+            "nosql"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
```

Comment:
The original SVG contains multiple paths and colors. Please create a `plain.svg` icon that follows our [SVG guidelines](https://github.com/devicons/devicon/wiki/SVG-Standards) and use that new icon for the font version,